netbench/cdk/src/main/java/com/aws/PeeringConnectionStack.java

```diff
@@ -0,0 +1,51 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.aws;
+
+import software.constructs.Construct;
+import software.amazon.awscdk.Stack;
+import software.amazon.awscdk.StackProps;
+import software.amazon.awscdk.services.ec2.CfnVPCPeeringConnection;
+import software.amazon.awscdk.services.ec2.Vpc;
+import software.amazon.awscdk.services.ec2.*;
+
+class PeeringConnectionStack extends Stack {
+    public PeeringConnectionStack(final Construct parent, final String id) {
+        this(parent, id, null);
+    }
+
+    public PeeringConnectionStack(final Construct parent, final String id, final PeeringStackProps props) {
```

Comment:
I think this could also be merged into the ClientServer stack? I suggest following something like this https://docs.aws.amazon.com/cdk/v1/guide/best-practices.html#best-practices-constructs.

Stacks are a unit of deployment, not application. I would group everything that needs to be deployed together into one stack to simplify your CDK code and it's deployment. 

IMO `PeeringConnectionStack` could be a construct